### DISCUSSION TITLE
Welcome to smart-launcher-v2 Discussions!

### DISCUSSION THREAD

**Discussion by vlad-ignatov** [smart-on-fhir/smart-launcher-v2#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by domusatlasnatasha** [comment]
Are CDS hook sandbox launches in scope for the smart-launcher-v2?

**Comment by katliger-h** [comment]
Today launch.smarthealthit.org seems to have lost its test patient data. Is it in the middle of a refresh?

**Reply by vlad-ignatov** [reply]
Which patient data is that. If it is something you inserted, then I'm afraid it may be expected. The data on these servers reset nightly and custom data is removed. You can insert a specific patient to test with, but you will have to do it every day.
